services/app-web/src/pages/SubmissionReleasedToState/ReleasedToState.tsx

```diff
@@ -151,14 +170,43 @@ const ReleasedToState = () => {
                                 as released after the approval letter has been
                                 released to the state.
                             </p>
-                            <FieldTextarea
-                                label="Provide an optional note"
-                                id="optionalNote"
-                                name="optionalNote"
-                                data-testid="optionalNote"
-                                aria-describedby="optionalNoteHelp"
-                                showError={false}
-                            />
+                            <Label
```

Comment:
**todo**: Let's move the `Label` and the <p> elements, that describe the input into the `FormGroup`.